### PR TITLE
device expand and delete

### LINKED ISSUE #11: T1: Blueprint and Device: Bin icon to delete device

### LINKED ISSUE #12: T1: Blueprint and Device: Click on device → Highlight device in ‘My Devices’ and blueprint expand device details

### PR DESCRIPTION
**Pull request by YiwenQiu**

### CODE REVIEW

File: src/components/pages/blueprints-and-devices/device-display/Device.jsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ import IconButton from '@material-ui/core/IconButton';
 import Typography from '@material-ui/core/Typography';
 
 import BinIcon from '../../../../assets/UI_component_svg/BinIcon';
+import React from 'react';
```
Comment (shandyk18) [inline src/components/pages/blueprints-and-devices/device-display/Device.jsx]: change to: `import { useState } from 'react';`

File: src/components/pages/blueprints-and-devices/device-display/Device.jsx [hunk 2]
```diff
@@ -33,31 +36,57 @@ const useStyles = makeStyles((theme) => ({
...
+
+   const removeCard = () => {
+      setRemove(true);
+   }
+   
+   if (removed) {
+      return false;
+   } else {
+      return (
+         <Card onClick={handleExpand} className={classes.cardContainer}  variant={outlined}>
```
Comment (shandyk18) [inline src/components/pages/blueprints-and-devices/device-display/Device.jsx]: delete extra space before variant

File: src/components/pages/blueprints-and-devices/device-display/Device.jsx [hunk 3]
```diff
@@ -33,31 +36,57 @@ const useStyles = makeStyles((theme) => ({
...
-               <IconButton>
-                  <BinIcon />
-               </IconButton>
-               <Button className={classes.activeBtn}>
-                  Active
-               </Button>
+
+   const [expanded, setExpanded] = React.useState(false);
+   const [outlined, setOutlined] = React.useState(null);
+   const [removed, setRemove] = React.useState(false);
```
Comment (shandyk18) [inline src/components/pages/blueprints-and-devices/device-display/Device.jsx]: condense these to `useState([value])`

**Review by shandyk18 — CHANGES_REQUESTED** [review]

File: src/components/pages/blueprints-and-devices/device-display/Device.jsx (hunk 3 above)
Comment (YiwenQiu) [reply src/components/pages/blueprints-and-devices/device-display/Device.jsx]: I have fixed all of them and pushed again. Do I need to create a new pull request? If so, how can I do that?

Comment (shandyk18) [reply src/components/pages/blueprints-and-devices/device-display/Device.jsx]: Any changes you make will be updated in the pull request, so there's no need to make another one!

**Review by shandyk18 — APPROVED** [review]

**Pull request merged**